File: SimpleTable/ClientApp/src/app/models/employee.ts
import { Department } from "./department";

export class Employee {
  constructor(
    public id: number,
    public firstName: string,
    public lastName: string,
    public fullName: string,
    public birthDate: Date,
    public gender: string,
    public isCertified: boolean,
    public phone: string,
    public email: string,
    public departmentId: number,
    public department: Department) { }
}

File: SimpleTable/Models/Employee.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace SimpleTable.Models
{
    public class Employee
    {
        [Key]
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        [NotMapped]
        public string FullName => FirstName + " " + LastName;
        public DateTime BirthDate { get; set; }
        public Gender Gender { get; set; }
        public bool IsCertified { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public int DepartmentId { get; set; }
        public Department Department { get; set; }
    }
}
